src/components/Drawer/Drawer.tsx

```diff
@@ -45,15 +40,15 @@ export interface Props {
    * Sets the drawer to open or close by default
    */
   isActive?: boolean;
-  /**
-   * Toggles the shadowed backdrop of the drawer on or off for flexibility
-   */
-  showBackdrop?: boolean;
   /**
    * Handler to be called when the drawer is being closed (by ESCAPE / clicking X / clicking outside)
    */
   onClose?: (e?: any) => void;
-}
+  /**
+   * CSS class names that can be appended to the Drawer window component that the Drawer components reside in.
+   */
+  windowClassName?: string;
+};
```

Comment:
I find the name and description here a bit confusing – I had to skip down and read the code to be sure I understood which class name went with which element. I'm actually wondering if the outer element should have the `windowClassName` or `drawerContainerClassName` or something like that (the `Modal` has `modalContainerClassName`, for example).

Reply:
Hmmm that makes sense, I'll rename to be less confusing